content/blog/pulumi-kubernetes-operator-2-0/index.md

```diff
@@ -0,0 +1,269 @@
+authors: ["eron-wright"]
+tags: ["Kubernetes", "Continuous-Delivery", "operators"]
+meta_desc: "Pulumi Kubernetes Operator 2.0: Horizontal Scaling, Multi-Tenancy"
+meta_image: operator.png
+---
+
+A few years ago we released the [Pulumi Kubernetes Operator](/blog/pulumi-kubernetes-operator-1-0/), a cloud-native way to manage and deploy cloud infrastructure using Pulumi from within your Kubernetes environment. We've heard your feedback about limitations related to scalability and isolation.
+Today, we're excited to make [2.0 beta 1](https://github.com/pulumi/pulumi-kubernetes-operator/releases/tag/v2.0-beta.1) of the Pulumi Kubernetes Operator available.
+We've put a new architecture in place to address these limitations and to provide new customization options. Let's dig in!
+
+## What is the Pulumi Kubernetes Operator?
+
+The Pulumi Kubernetes Operator defines a Kubernetes Custom Resource called `pulumi.com/v1/Stack`, which represents a Pulumi [stack](/docs/concepts/stack/). The Pulumi stack can be authored in any supported Pulumi language (TypeScript, Python, Go, .NET, Java, YAML) and can deploy and manage cloud infrastructure in any supported cloud (AWS, Azure, GCP, Kubernetes and 60+ additional cloud and SaaS providers). The Pulumi Kubernetes Operator triggers cloud deployments based on changes to the `Stack` Custom Resource or the resources it uses.
+
+As a result, the Pulumi Kubernetes Operator enables users to specify the desired state of their cloud infrastructure using resources managed directly in their Kubernetes cluster, which trigger creation, update and deletion of the detailed cloud infrastructure they need.
+
+<!--more-->
+
+For example, the following Kubernetes resource can be created to deploy a Pulumi program which provisions cloud infrastructure to manage NGINX provided in the target Git repo and commit, and future updates to the commit or config settings will deploy updates to the infrastructure.
+
+```yaml
+apiVersion: pulumi.com/v1
+kind: Stack
+metadata:
+  name: nginx-k8s-stack-production
+spec:
+  stack: acmecorp/nginx/production
+  projectRepo: https://github.com/acmecorp/pulumi-nginx
+  commit: 2b0889718d3e63feeb6079ccd5e4488d8601e353
+```
+
+The Pulumi Kubernetes Operator can also be used along with Pulumi’s [Kubernetes provider](/registry/packages/kubernetes/), which allows Pulumi programs to deploy resources to Kubernetes, either in the same cluster as the operator, or in another cluster. This can be used to package up complex Kubernetes infrastructure into more abstracted units, to mix Kubernetes and cloud infrastructure into a single unit of deployment, to define how application workloads are deployed into a cluster, or for a wide variety of additional use cases. The Pulumi Kubernetes provider has full support for the Kubernetes API available in every Pulumi language, including support for [Helm](/blog/full-access-to-helm-features-through-new-helm-release-resource-for-kubernetes/).
```

Comment:
Not sure if we want to refer to it as "Operator" or "Pulumi Kubernetes Operator" throughout.

```suggestion
The Pulumi Kubernetes Operator can also be used along with Pulumi’s [Kubernetes provider](/registry/packages/kubernetes/), which allows Pulumi programs to deploy resources to Kubernetes -- either in the same cluster as the Operator, or in another cluster. This can be used to package up complex Kubernetes infrastructure into more abstracted units, to mix Kubernetes and cloud infrastructure into a single unit of deployment, to define how application workloads are deployed into a cluster, or for a wide variety of additional use cases. The Pulumi Kubernetes provider has full support for the Kubernetes API available in every Pulumi language, including support for [Helm](/blog/full-access-to-helm-features-through-new-helm-release-resource-for-kubernetes/).
```